lib/active_record/connection_adapters/postgis/oid/spatial.rb

```diff
@@ -43,7 +46,9 @@ def self.parse_sql_type(sql_type)
               # otherType(a,b)
               geo_type = sql_type
             end
-            [geo_type, srid, has_z, has_m]
+            geographic = !(sql_type =~ /geography/).nil?
```

Comment:
```suggestion
            geographic = sql_type.match? /geography/
```

Reply:
Since I'm expecting `geographic` to be a boolean I originally had `!!sql_type.match? /geography/` but rubocop suggested my edit. It will probably work with `nil` as well though.

Reply:
You do not need the `!!` since `String#match` returns only true or false, and if `sql_type` can be null then you can use `/geography/.match? sql_type`

Reply:
Ahh yeah you're right. I was using `match` not `match?` so that's where the confusion came from. Will just use `match?`.